docs/collecting-data/collecting-from-own-applications/webview-tracker/index.md

```diff
@@ -0,0 +1,133 @@
+import Badges from '@site/src/components/Badges';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<Badges badgeType="Early Release"></Badges>
+```
+
+The [Snowplow WebView Tracker](https://github.com/snowplow-incubator/snowplow-webview-tracker) allows you to track Snowplow events from Web views in **mobile hybrid apps**.
+
+Hybrid apps are mobile apps that in addition to a native interface, provide part of the UI through an embedded Web view.
+Snowplow events are tracked from both the native code (e.g. written in Swift or Kotlin) as well as the Web view (in JavaScript).
+Our goal is to have both events tracked from the native code as well as the Web view, share the same session and appear as tracked with the same tracker.
+
+The WebView tracker should be integrated in Web apps used in Web views within native mobile apps.
+The tracker provides APIs to track Snowplow events.
+It forwards the events to the native app code to be tracked by the Snowplow mobile trackers ([iOS, Android tracker](../mobile-trackers/hybrid-apps/index.md), or [React Native](../react-native-tracker/hybrid-apps/index.md)).
+The diagram below shows the interaction of the WebView and mobile trackers in hybrid apps.
+
+```mermaid
+flowchart TB
+
+subgraph hybridApp[Hybrid Mobile App]
+
+    subgraph webView[Web View]
+        webViewCode[App logic]
+        webViewTracker[Snowplow WebView tracker]
+
+        webViewCode -- "Tracks events" --> webViewTracker
+    end
+
+    subgraph nativeCode[Native Code]
+        nativeAppCode[App logic]
+        nativeTracker[Snowplow iOS/Android/React Native tracker]
+
+        nativeAppCode -- "Tracks events" --> nativeTracker
+    end
+
+    webViewTracker -- "Forwards events" --> nativeTracker
+end
+
+subgraph cloud[Cloud]
+    collector[Snowplow Collector]
+end
+
+nativeTracker -- "Sends tracked events" --> collector
+```
+
+## Installation
+
+You may choose to install the tracker as an npm package or by loading it through an HTML script tag.
+
+<Tabs groupId="installation">
+  <TabItem value="npm" label="Using npm" default>
+
+To install the WebView tracker in your JavaScript or TypeScript app, add the npm package:
+
+```bash
+npm install --save @snowplow/webview-tracker
+```
+
+You will then be able to use the functions provided by the WebView tracker as follows:
+
+```typescript
+import { trackSelfDescribingEvent } from '@snowplow/webview-tracker';
+```
+
+In addition, you will need to install the iOS or Android tracker in your native code and configure and initialize a tracker (see the [mobile tracker docs][mobile-tracker-setup-docs]). Afterwards, you will be able to subscribe to and track the events from the WebView tracker in a Web view by calling `Snowplow.subscribeToWebViewEvents(webView)`.
+
+  </TabItem>
+  <TabItem value="tag" label="Using Snowplow tag">
+
+You may download the `sp.js` file from the Releases section Github, self-host it, and load to your page using the following tag:
```

Comment:
Is sp.js on any CDNs? If not might be worth chatting so we can get it on jsDelivr, makes testing easier.

Also:
```suggestion
You may download the `sp.js` file from the [Releases section on Github](...URL TO RELEASES...), self-host it, and load to your page using the following tag:
```

Reply:
No, it's not CDN, we can chat about how to deploy that later.

Also added the link to releases.